zkevm-circuits/src/evm_circuit/execution/stop.rs

```diff
@@ -21,28 +34,192 @@ impl<F: Field> ExecutionGadget<F> for StopGadget<F> {
     const EXECUTION_STATE: ExecutionState = ExecutionState::STOP;
 
     fn configure(cb: &mut ConstraintBuilder<F>) -> Self {
+        let code_size = cb.bytecode_length(cb.curr.state.code_hash.expr());
```

Comment:
```suggestion
        let code_length = cb.bytecode_length(cb.curr.state.code_hash.expr());
```
Suggestion to use the same names as in the spec.

Reply:
Fixed in [`f77e67d`](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/419/commits/f77e67d0e8f3d30d401b2a66a8d8a915e6348156).